packages/playground/assets/js/playground.js

```diff
@@ -242,4 +243,13 @@ function scrollSelectedMenuItemIntoView() {
       });
     }, 500);
   }
+}
+
+function createMetaTags() {
+  var metaTag = document.createElement("meta");
+  var controlName = document.querySelector(".control-header").innerText;
```

Comment:
this might break very easily if we change the document structure and don't notice the meta tags are wrong